src/Singolo/index.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const burgerButton = document.getElementById('burger');
+const mobileMenu = document.querySelector('.navigation-mobile');
+const mobileLogo = document.querySelector('.header__logo-mobile');
+
+burgerButton.addEventListener('click', () => {
+  burgerButton.classList.toggle('active');
+  mobileMenu.classList.toggle('active');
+  mobileLogo.classList.toggle('active');
+});
+
+
+const buttonPrev = document.querySelector('.slider__arrow--left');
+const buttonNext = document.querySelector('.slider__arrow--right');
+
+const slider = document.getElementById('slider');
+const slides = document.querySelectorAll('.slider__item');
+let CURRENTSLIDE = 0;
```

Comment:
In case CURRENTSLIDE is not a constant, rename this variable with camelCase. 
(Note: in case it's a constant. name should be: CURRENT_SLIDE)